app/helpers/application_helper.rb

```diff
@@ -7,6 +7,12 @@ def country_display_name(iso_code)
     iso_code
   end
 
+  def country_short_name(iso_code)
+    country = Country.find(iso_code)
+    return tag.span("#{country.emoji_flag} #{iso_code}", title: "country.name") if country
+    iso_code
+  end
+
```

Comment:
I'd consider making this a class method on the Country model or perhaps in its presenter concern.